src/popup/components/TransactionTagList.vue

```diff
@@ -4,11 +4,15 @@
     :class="{ dense }"
   >
     <TransactionTag
-      v-for="label in labels"
-      :key="label"
-      :tx-type="label"
-      class="title-tag"
       data-cy="label"
+      :tx="tx"
```

Comment:
Isn't that component also used  tx details?

Reply:
in TransactionInfo